### PR TITLE
JBPM-5410 - Add role assignments to case start page

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/kiegroup/jbpm-wb/pull/651

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac please have a look.

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java [hunk 1]
```diff
@@ -100,9 +101,15 @@ public CaseDefinitionSummary getCaseDefinition(final String serverTemplateId, fi
...
-        final CaseFile caseFile = CaseFile.builder().addUserAssignments(CASE_OWNER_ROLE, owner).build();
-        return client.startCase(containerId, caseDefinitionId, caseFile);
+    public String startCaseInstance(final String serverTemplateId, final String containerId, final String caseDefinitionId, final String owner,
+                                    final List<CaseRoleAssignmentSummary> roleAssignments) {
+        final CaseFile.Builder builder = CaseFile.builder();
+        builder.addUserAssignments(CASE_OWNER_ROLE, owner);
+        roleAssignments.stream().forEach( a -> {
+            a.getGroups().stream().forEach(g -> builder.addGroupAssignments(a.getName(), g));
+            a.getUsers().stream().forEach(u -> builder.addUserAssignments(a.getName(), u));
+        });
```
Comment (triceo) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java]: FYI Unless you actually need the streams,`Collection.forEach()` will do the same thing.

Comment (cristianonicolai) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java]: @triceo thanks, fixed.

**Comment by nmirasch** [comment]
+1

**Pull request merged**